source/testing_govuk_pay/index.html.md.erb

```diff
@@ -33,6 +33,11 @@ You can also:
 
 You cannot use 3D Secure with test accounts.
 
+If you use Stripe, you can request a test Stripe account by selecting your account in the [GOV.UK Pay admin tool](https://selfservice.payments.service.gov.uk/my-services). You can use a test Stripe account to see:
+
+- [transaction fees](/reporting/#psp-fees)
+- [how long it takes to receive money](/reporting/#checking-when-your-payment-service-provider-psp-took-a-payment)
```

Comment:
Cool, will update.

On the second point - do you mean this section: https://docs.payments.service.gov.uk/reporting/#checking-when-your-payment-service-provider-psp-took-a-payment)? I think the heading sounds right because you're checking the capture time, not when you actually received the payment. But if it feels wrong maybe best to raise another ticket so it doesn't hold up this work?

Reply:
I think this is related to the comments on the other ticket about th euser need for the settlement reporting API / payout report.  So I think we've prob covered that there.

I think the new text looks better on this page now